cmake/modules/FindDeprecated.cmake

```diff
@@ -99,6 +99,13 @@ if("PYTHON_PREFER" IN_LIST Deprecated_FIND_COMPONENTS)
   endif()
 endif()
 
+if("HEX_CONV" IN_LIST Deprecated_FIND_COMPONENTS)
+  # This code was deprecated after Zephyr v3.5.0
+  # The deprecated warning is produced by hex.cmake itself
+  list(REMOVE_ITEM Deprecated_FIND_COMPONENTS HEX_CONV)
+  include(${ZEPHYR_BASE}/cmake/hex.cmake)
+endif()
+
```

Comment:
This code doesn't make much sense now, as no one will be calling `find_package(Deprecated COMPONENTS HEX_CONV)`

```suggestion
```

Reply:
I'm not sure how things marked for deprecation are tracked... as I understood it, CMake components that are "_to be removed_" need to be listed here, so it's there as a reminder more than anything else.

I'll happily remove if you're okay with that.

Reply:
Ok, done - thanks!